File: 6.py
import math

print('Podaj a, b i c')
a = float(input())
b = float(input())
c = float(input())

ob = a + b + c # Ob oznacza obwod trojkata
p = ob / 2 # Litera p oznacza polowe obwodu trojkata
s = math.sqrt(p * (p - a) * (p - b) * (p - c)) # Wykorzystanie wzoru Herona do obliczenia pola

print('\nObwod tego trojkata to: ' + str(ob) + ', a pole to ' + str(s) + '.')

File: 6.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main()
{
    float a,b,c,ob,p,s;

    cout << "Podaj a, b i c" << endl;
    cin >> a;
    cin >> b;
    cin >> c;

    ob = a + b + c;
    p = ob / 2;
    s = sqrt(p * (p - a) * (p - b) * (p - c));

    cout << endl << "Obwod tego trojkata to: " << ob << " , a jego pole to " << s;
}
